### ISSUE TITLE
FileLogTraceListener.Append  Property behavior

### ISSUE THREAD

**Issue by MohammadHamdyGhanem** [dotnet/vblang#294]
I read this in Microsoft docs about the FileLogTraceListener.Append  Property:
https://docs.microsoft.com/ar-sa/dotnet/api/microsoft.visualbasic.logging.filelogtracelistener.append?view=netframework-4.7.2#Microsoft_VisualBasic_Logging_FileLogTraceListener_Append

> Property Value
> Boolean, with True indicating that the output is appended to the current file, and False indicating that output is written to a new file. The default setting for this property is True.

When I make FileLogTraceListener.Append false, new data is written to the same log file after erasing all previous content. No new file is created as said in docs!
Is this an error in docs, or a bug in FileLogTraceListener.Append?

**Comment by reduckted** [comment]
Interesting. It's hard to tell the intent behind the code because I'm only viewing it in ILSpy and there's no comments, but when it's creating a file, it _does_ check to see if the file is already being used by another `FileLogTraceListener` in the same AppDomain. But, of course, that only causes a new file to be created if you are creating multiple `FileLogTraceListeners` in the same AppDomain. If you close the application and start it again, it _will_ overwrite the last log file.

@KathleenDollard where should an issue like this be reported? This isn't technically a _VB language_ issue, but the code in question isn't open-source, so it doesn't seem appropriate to report a bug/feature request in the corefx repo.

@MohammadHamdyGhanem you could probably open a bug in the docs repo (or even make a PR to change the docs) because the documentation should probably clearly state that when `Append` is false, it will _overwrite_ the existing file instead of creating a new file. Here's the documentation in question: https://github.com/dotnet/dotnet-api-docs/blob/master/xml/Microsoft.VisualBasic.Logging/FileLogTraceListener.xml

**Comment by MohammadHamdyGhanem** [comment]
By the way, I also propsed to add the FileLogTraceListener class to corefx. I hope vb.net community vote to this:
https://github.com/dotnet/corefx/issues/29274

**Comment by KathleenDollard** [comment]
Are you working in Visual Studio? If so, feel free to use the feedback mechanism there, and it can be triaged through normal channels.

**Comment by KathleenDollard** [comment]
Thanks for letting us know about this. We determined that this is a documentation bug. When you are not appending it either:

* Creates a new file
* Overwrites the existing file. 

I submitted a PR to update the docs (which are Open Source, so anyone can improve). 

(I clicked the pencil in the upper right to edit - this took me to GitHub, where I had to click to edit again which forks and allows you to actually edit. At the bottom of that editor is a Save button. The last step is to create a PR back to the docs repo)

Fixed by: https://github.com/dotnet/dotnet-api-docs/pull/691
